packages/zoltan2/test/driver/driverinputs/kuberry/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Test problem that exposes a memory leak in MJ when num_global_parts==1
```

Comment:
Correct comment:
"Test problem for #7351 that exposes an out-of-bounds array access in MJ when rectilinear = 1"

Reply:
Fixed